csrankings-b.csv

```diff
@@ -364,7 +364,7 @@ Bernhard Seeger,University of Marburg,https://www.uni-marburg.de/de/fb12/arbeits
 Bernhard Sick,University of Kassel,https://www.uni-kassel.de/eecs/de/fachgebiete/ies/personen/bernhard-sick.html,sGAKnroAAAAJ
 Bernhard Steffen,TU Dortmund,http://ls5-www.cs.tu-dortmund.de/cms/de/mitarbeiter/prof/Bernhard_Steffen.html,YFGsf1YAAAAJ
 Bernhard Thalheim,University of Kiel,https://www.is.informatik.uni-kiel.de/thalheim,lkH3h9gAAAAJ
-Bernhard Thomaszewski,University of Montreal,http://www-labs.iro.umontreal.ca/~bernhard,Qtzmow0AAAAJ
+Bernhard Thomaszewski,ETH Zurich,https://inf.ethz.ch/people/person-detail.MTcwMTMx.TGlzdC8zMDQsLTg3NDc3NjI0MQ==.html,IDw2HJAAAAAJ

```

Comment:
This would be a better home page: https://n.ethz.ch/~bthomasz/ (feel free to submit a PR to change it, or not).